pkg/webui/components/select/index.js

```diff
@@ -131,6 +153,21 @@ class Select extends React.PureComponent {
     onBlur(event)
   }
 
+  @bind
+  loadOptions(inputValue, callback) {
+    const { loadOptions } = this.props
+    const requestResults = loadOptions(inputValue)
```

Comment:
There also needs to be some kind of debounce mechanism. Otherwise, a request is launched for each keystroke. Have a look at this:
https://github.com/JedWatson/react-select/issues/614#issuecomment-244006496